docker-compose.yml

```diff
@@ -89,3 +89,13 @@ cli:
 #     - web
 #   ports:
 #     - "4444:4444"
+
+# To use Solr, uncomment the service definition below and link the web service to the solr service.
+# solr:
+#   hostname: solr
+#   image: blinkreaction/drupal-solr:3.x
+#   ports:
+#     - "8983:8983"
```

Comment:
- "8983" should be enough for inter-container communication and to avoid conflicts between multiple solr containers launched on the same host